resources/schema.py

```diff
@@ -89,10 +90,15 @@ class Meta:
         interfaces = (relay.Node,)
 
 
-class HarborNode(graphql_geojson.GeoJSONType):
+class HarborFilter(django_filters.FilterSet):
+    servicemap_ids = django_filters.BaseCSVFilter(
```

Comment:
does it expected this format: `servicemapIds: "41234,41235,41236"` or this: `servicemapIds: ["41234", "41235", "41236"]`. I was thinking of the latter one tbh 🙂

Reply:
I was thinking of the same, but I think GrapQL (or at least [django-filters](https://django-filter.readthedocs.io/en/master/ref/filters.html)) doesn't support that format. All the filters with expect multiple values receive them with that format.

Reply:
combining [`BaseInFilter`](https://django-filter.readthedocs.io/en/master/ref/filters.html#baseinfilter) with [`CharFilter`](https://django-filter.readthedocs.io/en/master/ref/filters.html#charfilter)?

Reply:
The `BaseInFilter` "_only validates that the incoming value is comma-separated_". Also, "_The secondary filter is then used to validate the individual values_", so the `CharFilter` would only valditate that each of the CSV values are chars.

Also, the `BaseInFilter` only sets the `lookup_expr="in"`, so there's no additional benefit on using it. It's easier to have the `CSVFilter` and set the `lookup_expr` rather than adding another class just to combine both filters.

Reply:
FTR, GraphQL _does_ accept arrays as arguments, so it's a Django filters thing.